packages/switch/README.md

```diff
@@ -58,6 +58,44 @@ focus to the content.
 </div>
 ```
 
+## Sizes
+
+<sp-tabs selected="m" auto label="Size Attribute Options">
+
+<sp-tab value="s">Small</sp-tab>
+<sp-tab-panel value="s">
+
+```html
+<sp-switch size="s">Small</sp-switch>
+```
+
+</sp-tab-panel>
+
+<sp-tab value="m">Medium</sp-tab>
+<sp-tab-panel value="m">
+
+```html
+<sp-switch size="m">Medium</sp-switch>
+```
+
+</sp-tab-panel>
+
+<sp-tab value="l">Large</sp-tab>
+<sp-tab-panel value="l">
+
+```html
+<sp-switch size="l">Large</sp-switch>
+```
+
+</sp-tab-panel>
+
+<sp-tab value="xl">Extra Large</sp-tab>
+<sp-tab-panel value="xl">
+
+```html
+<sp-switch size="xl">Extra Large</sp-switch>
+```
+
```

Comment:
```suggestion

</sp-tab-panel>
</sp-tabs>
```

To prevent: https://bschmidt-switch-core-tokens--spectrum-web-components.netlify.app/components/switch/